src/data/drinkRecipes/autumn.ts

```diff
@@ -0,0 +1,78 @@
+import type { DrinkRecipe } from "types/DrinkRecipe"
+import { DrinkRecipeIngredientType } from "types/DrinkRecipe"
+
+const fakeImageUrl =
+  "https://www.liquor.com/thmb/gEsRZdHlTm7j2rVHC-_QkyxZUSw=/720x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/autumn-720x720-primary-357cede4ffa9487f93136f76b21d8591.jpg"
+
+const autumn: DrinkRecipe = {
+  name: "Autumn",
+  imageUrl: fakeImageUrl,
+  measuredIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Applejack",
+      unit: "oz",
+      unitAmount: 1.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Amontillado Sherry",
+      unit: "oz",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Pear Brandy",
+      unit: "oz",
+      unitAmount: 0.25,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Maple Syrup Grade A",
+      unit: "oz",
+      unitAmount: 0.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Apple Bitters",
+      unit: "Dash",
+      unitAmount: 3.0,
+      alternates: [],
+    },
+  ],
+  numberedIngredients: [],
+  portionlessIngredients: [],
+  garnishIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      ingredient: "Allspice",
+      suffix: "Tincture",
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Portionless,
+      prefix: "Dehydrated",
+      ingredient: "Pear",
+      suffix: "Crisp",
```

Comment:
Isn't pear crisp also totally different from pear? A quick google makes it look like it - again you tell me. If they are different, I think the ingredient itself should be "Pear Crisp"

Reply:
So - it's like a pair slice? I might use a different word. Maybe we shouldn't. I kinda want to. Up to you.